config/routes.rb

```diff
@@ -15,7 +15,7 @@
   get "apps/icon/:name(/:type(/:owner))" => "apps#icon", as: "app_icon", defaults: { type: "sys" }
 
   #FIXME: undo when ready to deploy app sharing to production, remove?
-  if ENV['OOD_APP_SHARING'].present?
+  if ENV['OOD_APP_DEVELOPMENT'].present?
```

Comment:
1. this should probably be NavConfig.show_develop_dropdown instead
2. the routes need to be split up into two groups, one for app sharing, and one for app development, and the separate flags should enable both; the way this PR currently is, OOD_APP_SHARING only does one thing: show a link in the develop dropdown. (this is wrong)